App/View/profile.php

```diff
@@ -43,4 +63,11 @@
     </div>
 </div>
 </div> 
-</div>
\ No newline at end of file
+</div>
+
+<div class="link">
+    <p1>Linked With Github</p1> &nbsp <img src="https://github.githubassets.com/images/modules/logos_page/GitHub-Mark.png" style=”width:40px;height:20px;">  
+</div>
+<div class="link"> 
+    <p2>Linked With Discord</p2> &nbsp <img src="https://preview.redd.it/s9biyhs4lix61.jpg?auto=webp&s=c600a95eff95b3e9406a8b913c6aa3988b5e3a8b" style=”width:40px;height:20px;">   
```

Comment:
Adding Style on HTML is really bad practice, use CSS instead

Reply:
Not sure how to do CSS spacing so I've just removed the HTML styling and left the result as is.